docs/deploying/wsgi-standalone.rst

```diff
@@ -123,8 +123,8 @@ If your httpd is not providing these headers, the most common setup invokes the
 host being set from ``X-Forwarded-Host`` and the remote address from
 ``X-Forwarded-For``::
 
-    from werkzeug.contrib.fixers import ProxyFix
-    app.wsgi_app = ProxyFix(app.wsgi_app)
+    from werkzeug.middleware.proxy_fix import ProxyFix
+    app.wsgi_app = ProxyFix(app.wsgi_app, x_proto=1)
```

Comment:
Also need to set `x_host=1` based on the server config above.

Reply:
Thanks for pointing out, I'll add that in

Reply:
I found that `middleware.proxy_fix` is only appeared above 0.15, so I think document may contains WARNING for werkzeug version.

Reply:
@linnil1 okay so you reckon we should add in a warning just below the sample usage that it only works above 0.15? Is it worth documenting the previous usage alongside that or is there no point?

Reply:
I have no idea for adding it or not.
Please ask @davidism.

Reply:
Is this okay to merge @davidism or do you reckon anything else needs adding/changing?